wenet/transformer/convolution.py

```diff
@@ -79,14 +81,15 @@ def __init__(self,
             self.norm = nn.LayerNorm(channels)
```

Comment:
The argument  of `BatchNorm1d` or `LayerNorm` maybe `channels if use_glu else 2 * channels`. 
In addition, the argument  `groups` of `depthwise_conv` maybe `channels if use_glu else 2 * channels` too.